classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :student
+
```

Comment:
- [ ] I suggest that you make this `attr_reader: students` not `student` and remember to change it anywhere you used it you will have an `@students = [ ]` then you will push each individual `student`